src/System.Runtime.Serialization.Formatters/tests/PlatformExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    internal static class PlatformExtensions
+    {
+        public static bool IsNetfxPlatform(this TargetFrameworkMoniker targetFrameworkMoniker)
+        {
+            switch (targetFrameworkMoniker)
+            {
+                case TargetFrameworkMoniker.netfx461:
+                case TargetFrameworkMoniker.netfx471:
+                case TargetFrameworkMoniker.netfx472:
+                    return true;
+            }
+
+            return false;
+        }
+
+        public static int GetPlatformIndex(this TypeSerializableValue[] blobs)
+        {
+            bool IsNetfx472PatchedOrNewer()
```

Comment:
I will remove that check as soon as all CI machines are patched. I don't wanna start versioning unpatched netfx blobs. This is just an exception to unblock the failing legs.